src/corehost/build.sh

```diff
@@ -148,12 +144,26 @@ while [ "$1" != "" ]; do
         --stripsymbols)
             __cmake_defines="${__cmake_defines} -DSTRIP_SYMBOLS=true"
             ;;
+        --rootdir)
+           shift
+           RootRepo=$1
```

Comment:
nit usually we do the shift after accessing the variable. not sure if this works.

Reply:
That's accompanied by the difference between `$1` and `$2`. Either works, this just happens to be the file's style.